cmd/server/assets/login/register-phone.html

```diff
@@ -100,8 +107,62 @@
       let $submitPin = $('#sms-code-submit');
       let $resendPin = $('#sms-code-resend');
 
+      let $registeredDiv = $('#registered-div');
+      let $factors = $('#factors');
+
       let verId = ""
 
+      firebase.auth().onAuthStateChanged(function(user) {
+        if (!user) {
+          return
+        }
+
+        if (user.multiFactor.enrolledFactors.length > 0) {
+          $skip.text("Account settings");
+          $skip.attr("href","/account");
+
+          for (i = 0; i < user.multiFactor.enrolledFactors.length; i++) {
+            let factor = user.multiFactor.enrolledFactors[i];
+            let li = $('<li class="list-group-item"></li>');
```

Comment:
It's usually convention to name jQuery variables prefixed with a `$`, so this would be `$li`.

Reply:
It's also _much_ faster to build a blank DOM node and then call `addClass`. As written, jquery has to parse the entire string and build the class list.